packages/module/src/Feedback/Feedback.scss

```diff
@@ -69,20 +69,20 @@
 }
 
 .chr-c-feedback-email {
-  margin-top: var(--pf-v5-global--spacer--lg);
+  margin-top: var(--pf-t--global--spacer--lg);
   overflow: hidden;
 }
 
 .chr-c-feedback-image {
-  background-color: var(--pf-v5-global--palette--black-150);
+  background-color: var(--pf-t--global--background--color--secondary--default);
 }
 
 .chr-c-feedback-panel {
-  margin-bottom: var(--pf-v5-global--spacer--lg);
+  margin-bottom: var(--pf-t--global--spacer--lg);
   &__body {
     padding-bottom: 4px;
     padding-top: 4px;
-    border-bottom: 1px solid var(--pf-v5-global--palette--black-500);
+    border-bottom: 1px solid var(--pf-t--global--border--color--nonstatus--gray--default);
```

Comment:
I also didn't see an exact match here, but this whole class appears to be unused, can it be removed @dlabaj?

Reply:
@wise-king-sullyman  Yes we can remove it.  I couldn't find it anywhere either.